app/src/components/posts/PostListItem.vue

```diff
@@ -55,13 +55,13 @@
       <v-card-text>
         <template v-for="column in columns">
           <div v-if="post[column.key]" :key="column.key">
-            <h3>{{ column.label }}</h3>
-            <p v-if="column.link">
+            <span>{{ column.label }}</span>
+            <span v-if="column.link">
               <a :href="post[column.link]" target="_blank">
                 {{ post[column.key] }}
               </a>
-            </p>
-            <p v-else v-html="post[column.key]" />
+            </span>
+            <spanZz v-else v-html="post[column.key]" />
```

Comment:
First of all: Sorry, I haven't been notified about your approval request..

The changes look fine to me, except for this line - is this intended?

Reply:
good catch! This was not intended and is fixed now!